bitrise.yml

```diff
@@ -0,0 +1,172 @@
+
+      Next steps:
+      - Check out [Getting started with iOS apps](https://devcenter.bitrise.io/en/getting-started/getting-started-with-ios-apps.html).
+    steps:
+    - script@1:
+        title: Clean Directory Script
+        inputs:
+        - content: |-
+            #!/usr/bin/env bash
+            # fail if any commands fails
+            set -e
+            # make pipelines' return status equal the last command to exit with a non-zero status, or zero if all commands exit successfully
+            set -o pipefail
+            # debug log
+            set -x
+
+            # write your script here
+            if [ -d "xcov_report" ]
+            then
+              rm -r xcov_report
+            fi
+            if [ -d "result" ]
+            then
+              rm -r result
+            fi
+            if [ -d ".output" ]
+            then
+              rm -r .output
+            fi
+            # or run a script from your repository, like:
+            # bash ./path/to/script.sh
+            # not just bash, e.g.:
+            # ruby ./path/to/script.rb
+    - git-clone@6: {}
+    - cache-pull@2: {}
+    - cocoapods-install@2: {}
+    - script@1:
+        inputs:
+        - content: |-
+            #!/usr/bin/env bash
+
+            set -e
+            set -x
+
+            # write your script here
+
+            xcrun xcodebuild -workspace WeatherToday.xcworkspace -scheme WeatherToday -sdk iphonesimulator -destination 'platform=iOS Simulator,name=iPhone 12 Pro,OS=15.2' -derivedDataPath '.output' test
+
+            mkdir result
+
+            find .output -iname '*.xcresult' |  xargs  -I _ cp -R _ result
+
+            find result -iname '*.xcresult' |  xargs  -I _ mv _ result/WeatherTestResult.xcresult
+        title: Script For Xcode Build Test And Gather Output
+    - danger@2:
+        inputs:
+        - github_api_token: "$DANGER_GITHUB_API_TOKEN"
+    - cache-push@2: {}
+    - deploy-to-bitrise-io@2: {}
+meta:
+  bitrise.io:
+    stack: osx-xcode-13.2.x
+    machine_type_id: g2.8core
+app:
+  envs:
+  - opts:
+      is_expand: false
+    BITRISE_PROJECT_PATH: WeatherToday.xcworkspace
+  - opts:
+      is_expand: false
+    BITRISE_SCHEME: WeatherToday
+  - opts:
+      is_expand: false
+    BITRISE_DISTRIBUTION_METHOD: development
+  - opts:
+      is_expand: false
+    BITRISE_DEPLOY_DIR: result
+  - opts:
+      is_expand: false
+    BITRISE_XCRESULT_PATH: result
+  - opts:
+      is_expand: false
+    BITRISE_XCODE_TEST_RESULT: result
+  - opts:
+      is_expand: false
+    BITRISE_APLLE_DEV_TEAM_ID: 4TWS7E2EPE
+  - opts:
+      is_expand: false
+    FASTLANE_MATCH_GIT_URL: https://github.com/nimblehq/match-certificates.git
+  - opts:
+      is_expand: false
+    APP_BUNDLE_ID: com.weatherToday.fastlanetest
+  - opts:
+      is_expand: false
+    FASTLANE_MATCH_GIT_BRANCH: master
+  - opts:
+      is_expand: false
+    FASTLANE_MATCH_PASSWORD: Pk8UiEqEFeaqTKknbHQDWezT
```

Comment:
Please use [Secret](https://devcenter.bitrise.io/en/builds/secrets.html) for sensitive information.

Reply:
updated in this [commit](https://github.com/nimblehq/ci-cd-sample-ios/pull/24/commits/f929d2482b81136e10768dcebc953b3d50315e5e)